gfx/NeoDays/pngs_tiles_10x10/items/toilet_paper_0.json

```diff
@@ -0,0 +1 @@
+{"id": ["toilet_paper"], "fg": ["toilet_paper"], "rotates": false, "bg": []}
```

Comment:
All the jsons that associated one sprite with one id are unecessary. Here for exemple just name the sprite toilet_paper and remove the json. Same for the other json with only one id to one sprite

Reply:
Done; is there some new version of ``compose.py`` or argument that handles it like that? The version I've got seems to require them by default.

Reply:
yes use the --use-all argument